drivers/dht/include/dht_params.h

```diff
@@ -40,38 +40,31 @@ extern "C" {
 #ifndef DHT_PARAM_PULL
 #define DHT_PARAM_PULL              (GPIO_IN_PU)
 #endif
-
-#define DHT_PARAMS_DEFAULT          {.pin     = DHT_PARAM_PIN, \
-                                     .type    = DHT_PARAM_TYPE, \
-                                     .in_mode = DHT_PARAM_PULL}
+#ifndef DHT_PARAMS
+#define DHT_PARAMS                  { .pin     = DHT_PARAM_PIN,  \
+                                      .type    = DHT_PARAM_TYPE, \
+                                      .in_mode = DHT_PARAM_PULL }
+#endif
+#ifndef DHT_SAULINFO
+#define DHT_SAULINFO                { .name = "dht" }
+#endif
 /**@}*/
 
 /**
  * @brief   Configure DHT devices
  */
 static const dht_params_t dht_params[] =
 {
-#ifdef DHT_PARAMS_BOARD
-    DHT_PARAMS_BOARD,
-#else
-    DHT_PARAMS_DEFAULT,
-#endif
+    DHT_PARAMS
 };
 
-/**
- * @brief   Get the number of configured DHT devices
- */
-#define DHT_NUMOF       (sizeof(dht_params) / sizeof(dht_params[0]))
-
-#ifdef MODULE_SAUL_REG
 /**
  * @brief   Allocate and configure entries to the SAUL registry
  */
-static const saul_reg_info_t dht_saul_reg_info[] =
+static const saul_reg_info_t dht_saul_info[] =
```

Comment:
`dht_saul_reg_info` is still referenced [here](https://github.com/RIOT-OS/RIOT/pull/8680/files#diff-81ccfd4c221c8991f7bbd00562b5dde9R70). Why is it still compiling?

Reply:
Good catch, this is leftover.

> Why is it still compiling?

Because it's not built by the CI. This is a known [issue/limitation](#6405)